src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -551,4 +635,52 @@ private void runWithRetries(ContextualRunnable runnable, TableBlob blob, String
       throw e;
     }
   }
+
+  private long getLatestSequenceNum(TableId tableId) throws InterruptedException, DeltaFailureException {
+    RetryPolicy<Object> retryPolicy = new RetryPolicy<>()
+      .withMaxDuration(Duration.of(context.getMaxRetrySeconds(), ChronoUnit.SECONDS))
+      // maxDuration must be greater than the delay, otherwise Failsafe complains
+      .withBackoff(Math.min(61, context.getMaxRetrySeconds()) - 1, 120, ChronoUnit.SECONDS)
```

Comment:
It looks like when maxRetrySecs are less than 60s, we will end up with retry just once? Should we consider retrying more times? maybe at least 2 or 3?

Reply:
In this case we are executing bigquery operations, which means the most likely cause of failure is some sort of misconfiguration or permissions issue, which are unlikely to be resolved without manual intervention. So if the user has set a low retry timeout, I think it's fine if we don't do many attempts.

Reply:
agree. makes sense for me